content/docs/tasks/security/health-check/index.md

```diff
@@ -22,14 +22,11 @@ this feature is not needed if the production setup is not using the
 
 ## Before you begin
 
-To complete this task, you can install Istio using one of the following paths:
+To complete this task, you can install Istio using **one of** the following paths:
 
 * To setup Istio without using Helm, follow the instructions in the
-  [Kubernetes installation guide](/docs/setup/kubernetes/install/kubernetes/). Remember to enable global mutual TLS with:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
-    {{< /text >}}
+  [installation guide](/docs/setup/kubernetes/install/kubernetes/).
+  Install Istio with the **strict mTLS mode** enabled.
```

Comment:
Can we list the commands here so people don't have to jump around?

Reply:
Several different docs are sharing the same content. I will see if that's possible and do it in a separate PR. Thanks.

Reply:
Please expand mTLS to mutual TLS

Reply:
Note that we have support for boilerplate snippets in the site infra to consolidate repeated text:

https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#embedding-boilerplate-text

Reply:
Thank you @geeknoid I will create a separate PR to organize the shared contents into the boilerplate snippets :)

Reply:
its pretty clear boiilerplate is needed for the various install cases.  We are not going to get that done today.  As far as I know, the documentation WG is going to write some type of SRE guide.  I think that boilerplate work can be lumped in with he SRE guide.  The duplication of all of the install all over the docs is mildly annoying, but more problematic is the maintenance of that duplicated install.

For this path, line 30-32 seem like the best approach (install without helm) while we take a longer term approach to simplifying the docs via the SRE guide.

Reply:
@sdake How is the SRE guide going? Does it cover the boilerplate work?

Reply:
@myidpt We are in the early planning stages of the SRE guide rework. We will need to identify and create boilerplate content as part of that effort for sure but like @sdake mentioned, that should be done at a later date.